buildscripts/kokoro/unix.sh

```diff
@@ -86,7 +88,7 @@ fi
 LOCAL_MVN_TEMP=$(mktemp -d)
 # Note that this disables parallel=true from GRADLE_FLAGS
 if [[ -z "${ALL_ARTIFACTS:-}" ]]; then
-  if [[ $ARCH == "aarch_64" ]]; then
+  if [ "$ARCH" = "aarch_64" ] || [ "$ARCH" = "ppcle_64" ]; then
```

Comment:
We prefer `[[` over `[`. This is best written as `[[ "$ARCH" = "aarch_64" || "$ARCH" = "ppcle_64" ]]`